src/sentry/middleware/integrations/integration_control.py

```diff
@@ -1,100 +1,48 @@
 from __future__ import annotations
 
 import logging
-import re
-from typing import Callable, Mapping, Type
+from typing import Callable, List, Type
 
 from django.http import HttpRequest, HttpResponse
 
-from sentry.silo import SiloMode
-
-from .parsers import (
-    BitbucketRequestParser,
-    BitbucketServerRequestParser,
-    GithubEnterpriseRequestParser,
-    GithubRequestParser,
-    GitlabRequestParser,
-    JiraRequestParser,
-    JiraServerRequestParser,
-    MsTeamsRequestParser,
-    SlackRequestParser,
-    VstsRequestParser,
+from sentry.middleware.integrations.classifications import (
+    BaseClassification,
+    IntegrationClassification,
+    PluginClassification,
 )
-from .parsers.base import BaseRequestParser
+from sentry.silo import SiloMode
 
 logger = logging.getLogger(__name__)
 
-ACTIVE_PARSERS = [
-    BitbucketRequestParser,
-    BitbucketServerRequestParser,
-    GithubEnterpriseRequestParser,
-    GithubRequestParser,
-    GitlabRequestParser,
-    JiraRequestParser,
-    JiraServerRequestParser,
-    MsTeamsRequestParser,
-    SlackRequestParser,
-    VstsRequestParser,
-]
+ResponseHandler = Callable[[HttpRequest], HttpResponse]
```

Comment:
this should be `HttpResponseBase` (the typical signature that django uses for view functions)